packages/svelte-api-reference/src/ApiReference.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+  import { onMount } from 'svelte'
+  import type { ReferenceConfiguration } from '@scalar/api-reference'
+  import { svelteThemeCss } from './theme'
+
+  export let configuration: ReferenceConfiguration
+  let configString = ''
+  let contentString = ''
+
+  onMount(() => {
+    initializeApiReference().catch(error => {
+      console.error('Failed to initialize API reference:', error)
+    })
+  })
+
+  const validateSpec = (spec) => {
+    if (!spec?.content && !spec?.url) {
+      throw new Error(
+        '[@scalar/svelte-api-reference] You didn’t provide a spec.content or spec.url. Please provide one of these options.',
+      )
+    }
+  }
+
+  const applyDefaultCss = (config) => {
+    if (!config?.customCss && !config?.theme) {
+      config.customCss = svelteThemeCss
+    }
+  }
+
+  const updateApiScript = (apiScript) => {
+    apiScript.setAttribute('data-configuration', configString)
+    apiScript.textContent = contentString
+  }
+
+  const sanitizeConfigString = (config) => {
+    return JSON.stringify(config ?? {})
+      .split('"')
+      .join('&quot;')
+  }
+
+  const loadCdnScript = () => {
+    const cdnScript = document.createElement('script')
+    cdnScript.src = "https://cdn.jsdelivr.net/npm/@scalar/api-reference"
+    document.body.appendChild(cdnScript)
+  }
```

Comment:
Uh, pardon me but why are you loading a script from CDN? Why not use the `esm` provided by `@scalar`?